src/components/PlatformError.jsx

```diff
@@ -36,7 +36,7 @@ const PlatformError = (props) => {
     reason = reason || 'That\'s all we know';
 
     if (error instanceof WorkResponseError) {
-      reason = 'We had an error on our side while we were completing your request.';
+      reason = error.message;
```

Comment:
Bubble up error message genereated in the API to the UI. Do we need this?

Reply:
I think it would be better to use our own message as before.

Ideally the api would only send a key or something that makes the UI able to identify it instead of exactly the error message we need to show. 
I say this because I feel like in this case the api's responsibility is informing the UI about what happened while the UI's responsibility is formatting what happened so that the user understands it .

Reply:
No, the error is not useful to the user.

Reply:
reverted

Reply:
reverted